pages/common/pngquant.md

```diff
@@ -0,0 +1,36 @@
+# pngquant
+
+> PNG converter and lossy image compressor.
+> More information: <https://pngquant.org/>.
+
+- Compress a PNG as much as possible and write result to a new file:
+
+`pngquant {{path/to/file.png}}`
+
+- Compress a PNG and override original:
```

Comment:
There are multiple fixes available: please chose the most appropriate one.

```suggestion
- Compress a specific PNG and override original:
```

```suggestion
- Compress a specific .png and override original:
```

The same issue as above.